client/verta/verta/_deployment/endpoint.py

```diff
@@ -32,10 +34,16 @@ def _create(cls, conn, conf, workspace, path, description=None):
 
     @classmethod
     def _create_json(cls, conn, workspace, path, description=None):
+        input = {}
```

Comment:
I would prefer `data` instead of `input` because the Client uses `data` everywhere else, but this is fine for now.